apps/meteor/server/services/omnichannel/service.ts

```diff
@@ -60,4 +62,32 @@ export class OmnichannelService extends ServiceClassInternal implements IOmnicha
 		const currentMonth = moment.utc().format('YYYY-MM');
 		return room.v?.activity?.includes(currentMonth) || !(await License.shouldPreventAction('monthlyActiveContacts'));
 	}
+
+	async mergeContacts(contactId: string, channel: ILivechatContactChannel): Promise<ILivechatContact | null> {
```

Comment:
Should we move this to under `overwriteClassOnLicense` too? wdyt @pierre-lehnen-rc ?

Reply:
I believe that would be better, since the merging feature is enterprise only

Reply:
Great, thanks! I moved it to EE too